Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.scss

```diff
@@ -19,6 +19,15 @@ mat-card-header {
   }
 }
 
+.blur {
+  background-size: cover;
+  -webkit-filter: blur(4px);
+  -moz-filter: blur(4px);
+  -ms-filter: blur(4px);
+  -o-filter: blur(4px);
+  filter: blur(4px) grayscale(0.01);
+}
+

```

Comment:
Mhh, i still think that this is a cool idea but don't know what the @T-Systems-MMS/phonebook-developers think about. If this will merged we can try to use it site-wide.

Reply:
But either implement it app-wide or use the standard.

Reply:
So i would say remove it and make an issue where all dialogs got this effect